app/admin/sponsor.rb

```diff
@@ -75,7 +75,7 @@
         column :orphan_date_of_birth
         column :orphan_gender
         column 'Sponsorship began' do |_sponsorship|
-          _sponsorship.start_date.strftime("%m/%Y")
+          ApplicationHelper.instance_method(:format_month_year_date).bind(binding).call(_sponsorship.start_date)
```

Comment:
That looks painful. What do you think about letting AA live out its remaining days as is? Or creating a separate helper for it that will be deleted along with the rest of AA code when time comes?